lua/graph.lua

```diff
@@ -1999,18 +2092,24 @@ function nodes_proc (kv, nokeys)
     truck_tag = truck_tag or 0
     moped_tag = moped_tag or 0
     motorcycle_tag = motorcycle_tag or 0
+    golf_cart_tag = golf_cart_tag or 0
   end
 
   local emergency_tag --implies nil
   if kv["access"] == "emergency" or kv["emergency"] == "yes" or kv["service"] == "emergency_access" then
-     emergency_tag = 16
+    emergency_tag = 16
   end
 
   --do not shut off bike access if there is a highway crossing.
   if bike_tag == 0 and kv["highway"] == "crossing" then
     bike_tag = 4
   end
 
+  --do not shut off golf cart access if there is a highway crossing.
+  if golf_cart_tag == 0 and kv["highway"] == "crossing" then
+    golf_cart_tag = 2048
+  end
+
```

Comment:
This is something to look into.

Make a gurka test to check if this is a problem for ones not marked as crossing. It might be deleteable.

Reply:
OK, I did a bit of research and testing here...

1. Gurka test (crossing a road where golf carts are not allowed; will be committed shortly) seems to pass just fine if I remove this logic.
2. I think I said in our last meeting that I assumed this was a direct translation of a business requirement. Looking over it again though, I realized I just copied it directly above from the same logic for bicycles ;) I will _not_ touch the bicycle behavior in this PR, but maybe this is room for simplification in the future I guess.
3. Confirmed this only applies at the node level in OSM (just a few hundred invalid way tags) + checked the area quickly for odd looking tags with Overpass Turbo. Shouldn't cause any issues.
4. I did some smoke testing with random routes in the areas we're serving in GA and FL and it doesn't look like it broke anything.

So, after all that, I went ahead and deleted this.